.github/workflows/edge.yml

```diff
@@ -153,92 +284,25 @@ jobs:
           name: 'trivy-results-${{ matrix.image }}.sarif'
           path: 'trivy-results-${{ matrix.image }}.sarif'
         if: always()
-    strategy:
-      matrix:
-        image: [debian, alpine, opentracing, ubi]
 
   smoke-tests:
     name: Smoke Tests
-    runs-on: ${{ matrix.os }}
+    runs-on: ubuntu-20.04
     needs: [binary, unit-tests]
-    if:
-      github.event.pull_request.head.repo.full_name == 'nginxinc/kubernetes-ingress' ||
```

Comment:
Wasn't this here to prevent builds from forks? We don't need it anymore?

Reply:
I left if for here to skip nightly, but I can change it to just pull requests and push events